client/packages/invoices/src/OutboundShipment/DetailView/AddFromScannerButton.tsx

```diff
@@ -13,12 +13,12 @@ import {
 } from '@openmsupply-client/common';
 import { useOutbound } from '../api';
 import { isOutboundDisabled } from '../../utils';
-import { Draft } from '../..';
+import { ScannedBarcode } from '../../types';
 
 export const AddFromScannerButtonComponent = ({
   onAddItem,
 }: {
-  onAddItem: (draft?: Draft) => void;
+  onAddItem: (scannedBarcode?: ScannedBarcode) => void;
```

Comment:
updated the onOpen logic to explicitly handle opened via barcode vs open via item click